doc/whats_new/v1.3.rst

```diff
@@ -36,6 +36,12 @@ Changelog
     :pr:`123456` by :user:`Joe Bloggs <joeongithub>`.
     where 123456 is the *pull request* number, not the issue number.
 
+:mod:`sklearn.ensemble`
+.......................
+- |Fix|
```

Comment:
The changelog is for users to know changes were made. May you include a sentence describing what has changed in this PR?